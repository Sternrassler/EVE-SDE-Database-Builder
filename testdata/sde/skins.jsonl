{"skinID":1001,"internalName":"aurgoralight_federation_navy_aurgoralight","skinMaterialID":123,"typeID":587,"allowCCPDevs":0}
{"skinID":1002,"internalName":"aurgoralight_federation_navy_coercer","skinMaterialID":124,"typeID":588,"allowCCPDevs":0}
{"skinID":1003,"internalName":"aurgoralight_federation_navy_tristan","skinMaterialID":125,"typeID":606,"allowCCPDevs":1}
