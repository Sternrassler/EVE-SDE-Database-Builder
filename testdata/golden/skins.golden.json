[
  {
    "skinID": 1001,
    "internalName": "aurgoralight_federation_navy_aurgoralight",
    "skinMaterialID": 123,
    "typeID": 587
  },
  {
    "skinID": 1002,
    "internalName": "aurgoralight_federation_navy_coercer",
    "skinMaterialID": 124,
    "typeID": 588
  },
  {
    "skinID": 1003,
    "internalName": "aurgoralight_federation_navy_tristan",
    "skinMaterialID": 125,
    "typeID": 606
  }
]